inmemory/list_test.go

```diff
@@ -15,6 +15,8 @@
 package inmemory
 
 import (
+	"github.com/xujiajun/nutsdb"
+	"github.com/xujiajun/nutsdb/ds/list"
```

Comment:
Imports are organized in groups https://github.com/golang/go/wiki/CodeReviewComments#imports